### ISSUE TITLE
Light theme in v1.3 ? and how do enable it.

### ISSUE THREAD

**Issue by macdevign** [255kb/mockoon#100]
I saw a request on light theme request that has resolved in v1.3, however after download it ,there is no option to set to light theme.

Is there a setting to disable dark theme ?
 In in opinion, dark theme is hard to get it right , often making the screen hard to read.  The worse, is using light color text against black background which cause eye fatigue.

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/6662383/52914380-735df780-3302-11e9-8b44-5496cd728d10.png">

**Comment by macdevign** [comment]
basically what i capture in the image look ok, but from screen perspective, the gray section is in fact black color due to using some soft screen protector to prevent glare from earlier macbook.

**Comment by 255kb** [comment]
Hi, due to limited time I have to work on this app I did not create a light theme. In the ticket you mentioned, I tried to improve the readability but did not work on a light theme. So the ticket was resolved but only because readability was improved a little bit. 
I must admit that a "normal" theme could be great but it won't be on the roadmap soon. 
I currently try to finish openAPI format import / export and also some performances improvements. 
I will let the ticket open and rename it so other people can vote for this feature 😃

**Comment by macdevign** [comment]
Great to hear that :}
Frankly, it will be great if you can consider putting this light theme as main priority for next improvement as I think if even the best features are integrated into this amazing tool, likely will be be hard to recommend it for a try given that the theme is hurting the eyes.  I still remember the Pixelmator case where for years it didn't have the light theme, which if it did, likely to have more users.

I think having 20% of features that are used in 80% most of the time will be good enough so as to make it easy to use, and not overly-complicated.

But still keep up the good work. I actually enjoy using mockoon even though it did hurt my eyes because it's features are good enough for use. Hope  that mockoon is not too feature-laden to the point that it is hard to use in the future, something that I notice in most of these types of tools :}.

thank for this amazing tool.

**Comment by 255kb** [comment]
I am glad that you like this tool, I spent a lot of time on it! 
I will try to add a light theme in the upcoming release, but please don't expect it too soon. I have a some upcoming family events that will keep me away from keyboard for a while. 😃 

Also I try to avoid adding too much features in this app. The reason I created Mockoon was because I needed something easy(ier) to use than other softwares out there. So I already refused some features. But if something get requested a lot (like OpenAPI import / export) I will definitely add it.

**Comment by axc450** [comment]
I can look into added this

**Comment by 255kb** [comment]
@axc450 Good idea. Would you mind joining the Spectrum community so we may chat a little bit? 😃 https://spectrum.chat/mockoon

**Comment by ricardo0624** [comment]
I like the light theme

**Comment by idevelop-app** [comment]
I vote for light theme as well, the app is great, just imagine how good it will be with a light theme!!

**Comment by git934** [comment]
I like the light theme
actual

**Comment by cirocorvino** [comment]
Please, add a light theme, personally I try difficult to focus on UI elements with dark theme on

**Comment by gwennael-jean** [comment]
+1
